src/utils/get.js

```diff
@@ -0,0 +1,17 @@
+const { net } = require('electron');
+
+// returns a promise that resolves to [statusCode, Buffer, headers]
+// [code, null, null] if request failed
+module.exports = async (url) => {
+  const request = new Request(url, {
```

Comment:
shouldn't this use `net.request`?

Reply:
You can review the current design choice I made. A `get` utility returning Promises that resolves to response data, and a more sophisticated `request` utility if more fine-grained control is needed.